apps/web/app/api/cron/report-usage/route.ts

```diff
@@ -0,0 +1,74 @@
+import { responses } from "@/app/lib/api/response";
+import reportUsage from "@formbricks/ee/billing/api/report-usage";
+import { NextResponse } from "next/server";
+import { headers } from "next/headers";
+import { CRON_SECRET } from "@formbricks/lib/constants";
+import { getMonthlyActivePeopleCount } from "@formbricks/lib/person/service";
+import { getTeamsWithPaidPlan } from "@formbricks/lib/team/service";
+import { getMonthlyResponseCount } from "@formbricks/lib/response/service";
+import { priceLookupKeys } from "@formbricks/ee/billing/utils/products";
+
+export async function GET(): Promise<NextResponse> {
+  const headersList = headers();
+  const apiKey = headersList.get("x-api-key");
+
+  if (!apiKey || apiKey !== CRON_SECRET) {
+    return responses.notAuthenticatedResponse();
+  }
+  try {
+    const teamsWithPaidPlan = await getTeamsWithPaidPlan();
+    for (const team of teamsWithPaidPlan) {
+      const stripeCustomerId = team.billing.stripeCustomerId;
+      if (!stripeCustomerId) {
+        continue;
+      }
+
+      let calculateResponses = team.billing.features.appSurvey.status !== "inactive";
+      let calculatePeople = team.billing.features.userTargeting.status !== "inactive";
+
+      if (!calculatePeople && !calculateResponses) {
+        continue;
+      }
+      let people = 0;
+      let responses = 0;
+
+      for (const product of team.products) {
```

Comment:
let's try to get the products of the teamId with `getProducts(team.id)` first an loop through this. To make this work we need to loop through all teams async.

e.g.:
```ts
const reportTeamUsage = async (team: TTeam) {
  const stripeCustomerId = team.billing.stripeCustomerId;
  ....
}

await Promise.all(teamsWithPaidPlan.map(reportTeamUsage));
```

Reply:
noicee! done!